collector/uname_linux_int8.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !nouname,linux,386 !nouname,linux,amd64 !nouname,linux,arm64 !nouname,linux,mips64 !nouname,linux,mips64le
+// +build !nouname,linux,386 !nouname,linux,amd64 !nouname,linux,arm64 !nouname,linux,mips64 !nouname,linux,mips64le !nouname,linux,mips !nouname,linux,mipsle
```

Comment:
Let's simplify the build tags. Multiple build tag constraints in multiple lines will be AND together, so please move `!nouname` into its own line. Maybe even `linux` as well.

```
// +build 386 amd64 ...
// +build linux
// +build !nouname
```